src/sedov/SedovDataGenerator.hpp

```diff
@@ -12,6 +12,16 @@ template <typename T, typename I>
 class SedovDataGenerator
 {
 public:
+
+    static constexpr double gamma         = 5.0/3.0;
+    static constexpr double r0            = 0.;
+    static constexpr double r1            = 0.5;
+    static constexpr double energytot     = 1.0;
+    static constexpr double width         = 0.10;
+    static constexpr double rho0          = 1.0;
+    static constexpr double ener0         = energytot / std::pow(M_PI,(3.0/2.0)) / 1.0 / std::pow(width,3);
```

Comment:
How is this not generating a compiler error in the CI? This should not compile because `std::pow` is not constexpr,
and in fact does not compile for me.

Reply:
I couldn't see this problem with make or cmake in my Ubuntu.
Do you use some additional flags for the compilation?

Maybe is a different compiler version. This is mine:
$ mpic++ --version
g++ (Ubuntu 11.2.0-7ubuntu2) 11.2.0

Reply:
As far as I can see on my system, the pow(x,y) function is taken from the include <math.h> at '/usr/x86_64-linux-gnu/include/bits/mathcalls.h'

$ cat /usr/x86_64-linux-gnu/include/bits/mathcalls.h
...
/* Return X to the Y power.  */
__MATHCALL_VEC (pow,, (_Mdouble_ __x, _Mdouble_ __y));
...

Reply:
It works with GCC and fails with clang. According to the C++17 standard, std::pow is not constexpr, so GCC is wrong and clang is right. This should be:
`static inline const double energ0 = ...`

Reply:
@jgphpc are we only testing with GCC?

Reply:
yes but I can add clang